plugin/openassetio_manager_bal/BasicAssetLibraryInterface.py

```diff
@@ -99,6 +114,11 @@ def initialize(self, managerSettings, hostSession):
         )
         self.__library = bal.load_library(library_path)
 
+        hostSession.logger().log(
+            hostSession.logger().Severity.kDebug,
+            f"Running with simulated query delay of '{self.__settings[bal.SETTINGS_KEY_SIMULATED_QUERY_LATENCY]}'ms",
```

Comment:
[consider] This message is super helpful - great idea. Is it worth aligning terminology with the setting name?
```suggestion
            f"Running with simulated query latency of '{self.__settings[bal.SETTINGS_KEY_SIMULATED_QUERY_LATENCY]}'ms",
```

Reply:
Yeah, agreed. ce87865759481d22d22793d208f212fbbfb958e8